hepcrawl/utils.py

```diff
@@ -85,3 +95,21 @@ def get_temporary_file(prefix="tmp_",
             pass
         raise e
     return filepath
+
+
+def get_mime_type(url):
```

Comment:
To test this function, we need to mock requests. If you wanna give it a try, take a look how to use the `responses` library in tests: https://github.com/getsentry/responses#response-body-as-string

Then you would call `get_mime_type` with the URL you have mocked and you test to make sure the function is doing the correct things. You can then test several things, such as:

* When `Content-Type` is given (this you can mock as part of responses.add), assert that the function returns it
* When `Content-Type` is not given, assert that an exception is raised. (https://pytest.org/latest/assert.html#assertions-about-expected-exceptions)
* When `url` is an empty string ("") or None, assert that an empty string is returned.